game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -53,6 +53,19 @@ object TeamDAO extends TableQuery(new Teams(_)) {
     )
   }
 
+  def updateScore(db: Database, team_id: Int, value: Int): Unit ={
+    val sql = sqlu"""update TEAMS
+                     set score = score + ${value}
+                     where id = ${team_id}"""
+    // TODO Log errors if there's any
```

Comment:
Yes, but what are we going to do with this value? Will it have failed because the `id` is incorrect or because the `database` is not responding?